### DISCUSSION TITLE
nswag issue with .net 6

### DISCUSSION THREAD

**Discussion by TweakStories** [jasontaylordev/CleanArchitecture#506] [Q&A]
Hello, 

When I try out the template I get this error: 

Severity    Code    Description    Project    File    Line    Suppression State
Error    MSB3073    The command "dotnet ".nuget\packages\nswag.msbuild\13.9.4\build\../tools/Net50/dotnet-nswag.dll" run nswag.json /variables:Configuration=Debug" exited with code -2147450730.    WebUI    

![image](https://user-images.githubusercontent.com/36763887/142846337-6c649fd6-8b96-43fe-aa03-217e111fd11b.png)

Did not change any of the code after creating a project. 

Does anyone know a fix for this? Thank you very much.

**Comment by lucianoStVieira** [comment]
I was having the same problem but it was about the Dependency Injection. I forgot to register the interface and the service at the container.

**Comment by tkomane** [comment]
@TweakStories I had the same issue on .NET 6. If you look at the error log you'll see that the nswag package has a dependency on .NET 5:

> Error MSB3073 The command "dotnet ".nuget\packages\nswag.msbuild\13.9.4\build../tools/**Net50**/dotnet-nswag.dll" run nswag.json /variables:Configuration=Debug" exited with code -2147450730. WebUI

I was able to run the project successfully after additionally installing .NET 5. I'm on Ubuntu so I used [these instructions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu), and specified aspnetcore-runtime-5.0 instead.

Hope that helps

**Comment by deceptiveSimplicity** [comment]
Use `$(NSwagExe_Net60)` instead of `$(NSwagExe_Net50)`
